packages/create-emotion-styled/types/index.d.ts

```diff
@@ -0,0 +1,101 @@
+// Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
+// TypeScript Version: 2.3
+
+import { Emotion, Interpolation as BaseInterpolation } from 'create-emotion';
+import React, { ComponentClass, ReactHTML, ReactSVG, Ref, SFC } from 'react';
+
+export interface ArrayInterpolation<Props> extends Array<Interpolation<Props>> {}
+export type FunctionInterpolation<Props> = (props: Props, context: any) => Interpolation<Props>;
+
+export type Interpolation<Props> =
+  | BaseInterpolation
+  | ArrayInterpolation<Props>
+  | FunctionInterpolation<Props>
+  ;
+
+export interface StyledOptions {
+  e?: string;
+  label?: string;
+  target?: string;
+  shouldForwardProp?: (name?: string) => boolean;
```

Comment:
This should be `(name: string) => boolean`, the name argument will always be there.